frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/inventory.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { DashboardCard } from '@console/internal/components/dashboard/dashboard-card/card';
+import { DashboardCardBody } from '@console/internal/components/dashboard/dashboard-card/card-body';
+import { DashboardCardHeader } from '@console/internal/components/dashboard/dashboard-card/card-header';
+import { DashboardCardTitle } from '@console/internal/components/dashboard/dashboard-card/card-title';
+import { FirehoseResource } from '@console/internal/components/utils';
+import { InventoryItem } from '@console/internal/components/dashboard/inventory-card/inventory-item';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import {
+  NodeModel,
+  PersistentVolumeClaimModel,
+  PersistentVolumeModel,
+} from '@console/internal/models';
+import {
+  getNodeStatusGroups,
+  getPVCStatusGroups,
+  getPVStatusGroups,
+} from '@console/internal/components/dashboard/inventory-card/utils';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+
+const cephStorageProvisiners = ['ceph.rook.io/block'];
+
+const getFilteredPVCsData = (pvcsData) =>
+  pvcsData.filter((pvc) =>
+    cephStorageProvisiners.includes(
+      _.get(pvc, 'metadata.annotations["volume.beta.kubernetes.io/storage-provisioner"]'),
+    ),
+  );
+
+const getFilteredPVsData = (pvsData) =>
+  pvsData.filter((pv) =>
+    cephStorageProvisiners.includes(
+      _.get(pv, 'metadata.annotations["pv.kubernetes.io/provisioned-by"]'),
+    ),
+  );
+
+const k8sResources: FirehoseResource[] = [
+  {
+    isList: true,
+    kind: PersistentVolumeModel.kind,
+    prop: 'pvs',
+  },
+  {
+    isList: true,
+    kind: NodeModel.kind,
+    prop: 'nodes',
+  },
+  {
+    isList: true,
+    kind: PersistentVolumeClaimModel.kind,
+    prop: 'pvcs',
+  },
+];
+
+const InventoryCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  React.useEffect(() => {
+    k8sResources.forEach((r) => watchK8sResource(r));
+    return () => {
+      k8sResources.forEach((r) => stopWatchK8sResource(r));
+    };
+  }, [watchK8sResource, stopWatchK8sResource]);
+
+  const nodes = _.get(resources, 'nodes');
```

Comment:
Do we need to use `_.get` here? You lose the type checking. I don't think these values will ever be null.

Reply:
https://github.com/openshift/console/pull/1914#discussion_r301803162

Reply:
`const nodes = _.get(resources, 'nodes');` could be deleted and then
`const nodesLoaded = _.get(resources.nodes, 'loaded');`
`const nodesData = _.get(resources.nodes, 'data', []) as K8sResourceKind[];`
That could work

Reply:
I agree with @rawagner, just `_.get` the stuff you actually need:

```ts
const nodesLoaded = _.get(resources.nodes, 'loaded');
const nodesData = _.get(resources.nodes, 'data', []) as K8sResourceKind[];
```